### PR TITLE
Kicad 5.1.5

### PR DESCRIPTION
**Pull request by zpuskas**

### CODE REVIEW

File: sci-electronics/kicad-i18n/kicad-i18n-5.1.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils
+
+DESCRIPTION="Electronic Schematic and PCB design tools GUI translations."
```
Comment (juippis) [inline sci-electronics/kicad-i18n/kicad-i18n-5.1.5.ebuild]: There's usually no dot in the DESCRIPTION.

File: sci-electronics/kicad/kicad-5.1.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,131 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Can switch to EAPI=7 when wxwidgets eclass also supports it
+EAPI=6
```
Comment (juippis) [inline sci-electronics/kicad/kicad-5.1.5.ebuild]: Whats holding the EAPI-7 upgrade? You can use `xdg_icon_cache_update` with EAPI-7 and drop gnome2-utils.eclass altogether. Also please leave one empty line below EAPI.

File: sci-electronics/kicad/kicad-5.1.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+WX_GTK_VER="3.0-gtk3"
+
+inherit check-reqs cmake-utils eapi7-ver eutils gnome2-utils python-single-r1 toolchain-funcs wxwidgets xdg-utils
+
+DESCRIPTION="Electronic Schematic and PCB design tools"
+HOMEPAGE="http://www.kicad-pcb.org"
```
Comment (juippis) [inline sci-electronics/kicad/kicad-5.1.5.ebuild]: `https://` when ever available.

File: sci-electronics/kicad/kicad-5.1.5.ebuild [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc examples github +ngspice occ +oce openmp +python"
+
+REQUIRED_USE="
+	python? ( ${PYTHON_REQUIRED_USE} )
+	?? ( occ oce )
+"
+
+COMMON_DEPEND="x11-libs/wxGTK:${WX_GTK_VER}[X,opengl]
```
Comment (juippis) [inline sci-electronics/kicad/kicad-5.1.5.ebuild]: Could you please sort this list to consistent style of proxy-maint (so it gets easier to maintain), ie unconditional deps before conditional ones, and both separately in alphabetical order.

**Comment by zpuskas** [comment]
@juippis I've addressed all your comments. I'm aware of the 

`scanelf: rpath_security_checks(): Security problem NULL DT_RUNPATH in /tmp/portage/sci-electronics/kicad-5.1.5/image/usr/lib64/python3.6/site-packages/_pcbnew.so`

problem, as well as the QA notice with `-Wstrict-aliasing`.

However after merging pcbnew works both integrated as well as standalone including the the Python console. Above issues seem to have popped up during yesterday's build, I'm not sure if EAPI7 enforces new checks, or something in the build toolchain has changed since.

I'd recommend merging this as is, with the caveat that it cannot be stabilized till both issues are resolved. Going through upstream's mailing list I've found that others have had similar issues before (the CMakeLists.txt is hacky) and whilte upstream is trying to fix the problem, it seems it's still far from perfect.  However the fact is that this problem existed already before the checks have caught it and there were no issues from the user side. I will follow up with upstream on this and we can also open Gentoo bugzilla trackers, while users on testing can test the application too.

**Pull request closed without merging**